### PR TITLE
added power of 2 code in php

### PR DESCRIPTION
**Pull request by abhijeet010304**
**Fixes issue:**

Created code to check a number is power of two or not.

**Changes:**

Added code to check a number is power of two or not  .

### CODE REVIEW

File: code/bit_manipulation/src/power_of_2/power_of_2.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+function is_Power($x)
```
Comment (arnavb) [inline code/bit_manipulation/src/power_of_2/power_of_2.php]: use camel case. Also beautify your code somewhere.

File: code/bit_manipulation/src/power_of_2/power_of_2.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ else
+    {
+		return "$x is power of 2";
+    }
+
+}
+
+print_r(is_Power(32)."\n");
+print_r(is_Power(31)."\n");
+?>
```
Comment (arnavb) [inline code/bit_manipulation/src/power_of_2/power_of_2.php]: Don't include a closing tag.

File: code/bit_manipulation/src/power_of_2/power_of_2.php (hunk 1 above)
Comment (abhijeet010304) [reply code/bit_manipulation/src/power_of_2/power_of_2.php]: Ok

File: code/bit_manipulation/src/power_of_2/power_of_2.php (hunk 2 above)
Comment (abhijeet010304) [reply code/bit_manipulation/src/power_of_2/power_of_2.php]: OK Sir I have corrected it please review!

**Pull request merged**